GOVERNANCE.md

```diff
@@ -0,0 +1,123 @@
+The CoreDash umbrella project is composed as a federation of individual projects,
+some independent and some interdependent, each of which focuses on some aspect
+of CoreDash. Our governance reflects this federated structure.
+
+CoreDash is a Linux Foundation project.
+
+## Individual Subproject Governance
+
+All active Maintainers of each [subproject](subproject.md), are members of that
+project's Maintainer Committee, which governs that project.  The subproject's
+Maintainer Committee is responsible for the following project governance
+activities:
+
+* Ensuring that the subproject creates and publishes regular releases;
+* Holding regular, subproject-wide discussions on issues and planning;
+* Making final decisions on subproject changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Identifying contributors who are candidates for moving into a maintainer role.
+
+Additionally, each project's Maintainer Committee will select, by majority vote, one
+representative to the Steering Committee to be replaced or
+renewed by the committee annually.
+
+Should a member of the Maintainer Committee cease being active in the subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Committee members, or a
+majority vote of the Steering Committee.
+
+## Adopter Council
+
+The Adopter Council exists to represent the needs of CoreDash users. New
+members can be added to the Adopter Council by a majority vote of the existing
+members.  
+
+The Adopter Council is responsible for the following project governance
+activities:
+
+* Review and provide feedback on new projects, roadmaps, and features.
+* Hold regular Adopter Council meetings open to all end users and other adopters.
+* Gather feedback from CoreDash adopters and share it broadly with the other 
+  subprojects and the Steering Committee. 
+
+The Adopter Council will have a chair who is responsible for coordinating
+Adopter Council activities. This person will be appointed by the Steering
+Committee and will be replaced or renewed by the committee annually.
+
+Should the Adopter Council Chair cease being active in the project,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a majority vote of the Steering Committee.
+
+## Steering Committee
+
+The overall CoreDash umbrella project is governed by a Steering
+Committee, which is selected as follows:
+
+* One Maintainer representative from each member subproject
+* One LF representative
+* One Adopter Council Chair
+* Three Community Representatives
+
+### Steering Committee Duties
+
+The Steering Committee is responsible for the following tasks, any of which may
+be delegated to a person or group selected by the committee:
+
+* Reviewing and deciding on new projects to add to CoreDash
+* Arbitrating inter-project disagreements
+* Ensuring that the LF Code of Conduct is enforced
```

Comment:
I dug into this one a bit more, and the LF projects seem to use the Contributor Covenant, but with each project adding it separately, rather than pointing to a central CoC the way most CNCF projects do. I updated the language and added a link to the CoC to make this clear.